jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -63,7 +71,8 @@ YUI.add('deployment-flow', function() {
         loggedIn: !!this.props.getAuth(),
         region: this.props.region,
         showChangelogs: false,
-        sshKey: null
+        sshKey: null,
+        termsAgreed: null
```

Comment:
It might be best to just leave it out of the initial state altogether. I'll see if that's possible.